Wiki/index.md

```diff
@@ -114,128 +110,100 @@ scientific computations instead of securing the blockchain.
-      - BTSABC: <https://bit.btsabc.org/#/market/OPEN.GRC_OPEN.BTC>
-      - Bitshares-Light: <https://bitshares.org/>
-      - RUDEX: <https://staging.rudex.org/#/market/OPEN.GRC_OPEN.BTC>
-      - Flyp.me: <https://flyp.me/>
-      - SouthXchange: <https://www.southxchange.com/Market/Book/GRC/BTC>
+      - [Txbit](https://txbit.io/Trade/GRC/BTC)
+      - [SouthXchange](https://www.southxchange.com/Market/Book/GRC/BTC)
 
 <!-- end list -->
 
   - Other Places To Trade:
-      - MarketGrid: <https://www.reddit.com/r/MarketGrid/>
-      - Gridcoin-OTC Telegram: <https://t.me/localgridcoins>
-      - Gridcoin-OTC IRC:
-        <https://kiwiirc.com/client/irc.snoonet.org:6697/#gridcoin-otc>
+      - [MarketGrid](https://www.reddit.com/r/MarketGrid/)
+      - [Gridcoin-OTC Telegram](https://t.me/localgridcoins)
+      - [Gridcoin-OTC IRC](https://kiwiirc.com/client/irc.snoonet.org:6697/#gridcoin-otc)
 
 <!-- end list -->
 
   - 3rd-Party wallets:
-      - CoinVault: <https://coinvault.io>
-      - Coinomi: <https://coinomi.com/>
+      - [Coinomi](https://coinomi.com/)
+      - [HolyTransaction](https://holytransaction.com/)
 
 <!-- end list -->
 
   - Forums:
-      - Cryptocurrencytalk:
-        <https://cryptocurrencytalk.com/forum/464-gridcoin-grc/>
-      - Cryptocurrencytalk macOS:
-        <https://cryptocurrencytalk.com/topic/13139-os-x-builds-feedback-bug-reporting/>
-      - Bitcointalk: <https://bitcointalk.org/index.php?topic=324118.0>
-      - Steemit: <https://steemit.com/created/gridcoin>
-      - Reddit: <https://reddit.com/r/gridcoin>
-      - Reddit (ru): <https://reddit.com/r/russiangridcoin>
-      - Rocket.Chat: <https://chat.gridcoin.io/channel/general>
-      - Gridcoin Telegram: <https://t.me/gridcoin>
-      - Discord: <https://discord.me/page/gridcoin>
-      - Slack:
-        <https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTUzMmY0YjdiNzYxYzQ0MDc3MGE1NjQ3Nzg2NWMzZTUzMjAwZjdhMWI1YWUzMDE4YzQyZjVjMjMzOTc1M2RmMmM/>
+      - [Cryptocurrencytalk](https://cryptocurrencytalk.com/forum/464-gridcoin-grc/)
+      - [Cryptocurrencytalk macOS](https://cryptocurrencytalk.com/topic/13139-os-x-builds-feedback-bug-reporting/)
+      - [Bitcointalk](https://bitcointalk.org/index.php?topic=324118.0)
+      - [Steemit](https://steemit.com/created/gridcoin)
+      - [Steemit Community](https://steemit.com/created/hive-161364)
+      - [Reddit](https://reddit.com/r/gridcoin)
+      - [Reddit (ru)](https://reddit.com/r/russiangridcoin)
+      - [Gridcoin Telegram](https://t.me/gridcoin)
+      - [Discord](https://discord.me/page/gridcoin)
+      - [Slack](https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTUzMmY0YjdiNzYxYzQ0MDc3MGE1NjQ3Nzg2NWMzZTUzMjAwZjdhMWI1YWUzMDE4YzQyZjVjMjMzOTc1M2RmMmM/)
       - IRC: #gridcoin (must be registered with NickServ) on Freenode
           or
           <https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin-help>
```

Comment:
Should probably change irc links to hyperlinked liked \[#gridcoin](https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin) and \[#gridcoin-help](https://kiwiirc.com/client/irc.freenode.net:6697/#gridcoin-help)